pkg/tests/tests.go

```diff
@@ -108,3 +117,35 @@ func resourceReviewbody() []byte {
 	}
 	return buff.Bytes()
 }
+
+func clusterAuthorizationsBody() []byte {
```

Comment:
@nimrodshn could you help me with this part. I have no idea what parameters values we need to use to build the body, or if we can build it here, maybe it has to be built individually for each request we want to try.

Reply:
@markturansky for save! He definitely knows better for the SDB endpoints.